learn2track/models/gru.py

```diff
@@ -143,10 +152,19 @@ def load(self, path):
 
     @classmethod
     def create(cls, path, **kwargs):
-        loaddir = pjoin(path, cls.__name__)
-        hyperparams = smartutils.load_dict_from_json_file(pjoin(loaddir, "hyperparams.json"))
-        hyperparams.update(kwargs)
-
-        model = cls(**hyperparams)
-        model.load(path)
+        try:
+            loaddir = pjoin(path, cls.__name__)
+            hyperparams = smartutils.load_dict_from_json_file(pjoin(loaddir, "hyperparams.json"))
+            hyperparams.update(kwargs)
+
+            model = cls(**hyperparams)
+            model.load(path)
+        except FileNotFoundError:
```

Comment:
Usually, instead of pointing to `experiments/my_experiment/`, I point directly to `experiments/my_experiment/training`